### DISCUSSION TITLE
An example of keyboard focus control described by a table

### DISCUSSION THREAD

**Discussion by gbishop** [gbishop/OS-DPI#12] [Ideas]
I'm just trying ideas. Check it out here: https://github.com/gbishop/OS-DPI/tree/access/thinking/key-access-test.

It only supports Tab for moving between components, F2 for entering and exiting a component and arrows for moving inside a component.

It would be neat if the actions could be broken down more. One function per is not that interesting...

**Comment by christineiym** [comment]
I am still immersing myself in the repository and ARIA guidelines, but here is a possible example with more actions, based on combining code from #10 with the initial key-access-test commit: https://github.com/christineiym/OS-DPI/tree/access/thinking/method-key-access-test
It does not completely work, as I need to allow input normally controlled with up/down arrows to be entered/edited, but it seems to work well with the keyboard focus control bindings.

**Comment by gbishop** [comment]
Interesting discussion here: https://github.com/mar10/fancytree/issues/709
